api_documentation/Core.postman_collection.json

```diff
@@ -238,7 +238,38 @@
 								{
 									"key": "container-id",
 									"type": "string",
-									"value": "20e22d75-338f-4b7d-b3d3-aafaeb1f8621"
+									"value": ""
+								}
+							]
+						},
+						"description": "Fetch container by ID."
+					},
+					"response": []
+				},
+				{
+					"name": "Query Files",
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    files(where: {\r\n        AND: [{file_name: \"like archive.json\"}]\r\n    }) {\r\n        id\r\n        \r\n    }\r\n}"
+						},
+						"url": {
+							"raw": "{{baseURL}}/containers/:container-id/query",
```

Comment:
What's the reasoning for not having "files" in the query path here? Is this querying content _within_ a file?

Reply:
Remember that files are stored differently than the nodes/edges. File storage right now is a standalone portion of the application so when we query via graphQL, we would query against "files" not the graph